ci/README.md

```diff
@@ -20,63 +20,74 @@ Available functionality:
 > engine) as defined in [`.github/workflows/_hack_make.yml`](../.github/workflows/_hack_make.yml).
 > Without this, you may hit unexpected errors or other weird behavior.
 
+## Developing after a fresh clone
+
+If you want to develop this module following a repository clone, remember to
+run the following one-off command:
+
+    cd ..
+    dagger develop --sdk go --source ci 
+
+This will re-create all the files required by your code editor - see
+`.gitignore` for a list of what they are.
+
 ## Tests
 
 Run all tests:
 
-    $ dagger call --source=. test all
+    dagger call --source=. test all
 
 Run a specific test (e.g. `TestModuleNamespacing`):
 
-    $ dagger call --source=. test custom --run="^TestModuleNamespacing" --pkg="./core/integration"
+    dagger call --source=. test custom --run="^TestModuleNamespacing" --pkg="./core/integration"
 
 ## Dev environment
 
 Start a little dev shell with dagger-in-dagger:
 
-    $ dagger call --source=. dev
+    dagger call --source=. dev
 
 ## Engine & CLI
 
 ### Linting
 
 Run the engine linter:
 
-    $ dagger call --source=. engine lint
+    dagger call --source=. engine lint
 
 ### Build the CLI
 
 Build the CLI:
 
-    $ dagger call --source=. cli file export --path=./dagger
+    dagger call --source=. cli file export --path=./dagger
 
 ### Run the engine service
 
 Run the engine as a service:
 
-    $ dagger call --source=. engine service --name=dagger-engine up --ports=1234:1234
+    dagger call --source=. engine service --name=dagger-engine up --ports=1234:1234
 
 Connect to it from a dagger cli:
 
-    $ export _EXPERIMENTAL_DAGGER_RUNNER_HOST=tcp://0.0.0.0:1234
-    $ dagger call -m github.com/shykes/daggerverse/hello@main hello
+    export _EXPERIMENTAL_DAGGER_RUNNER_HOST=tcp://0.0.0.0:1234
+    dagger call -m github.com/shykes/daggerverse/hello@main hello
```

Comment:
If we remove the `$` from here, there's no way to distinguish between the prompt and the output from it.

I know github has a way for copy-pasting code blocks, so if we want to do that, we need to remove the output from a command (or move it into a separate code block afterwards).

Reply:
I see what you mean. Adding a comment will fix it.